bahmni-standard/.env

```diff
@@ -180,3 +180,6 @@ SMS_COUNTRY_CODE=
 
 RESTORE_ARTIFACTS_PATH=./restore-artifacts
 RESTART_POLICY=unless-stopped
+
+IPD_TAG=ipd
```

Comment:
The image should be published under the `latest` tag instead of relying on the `ipd` tag. Please make the necessary changes in the `openmrs-ipd-frontend` workflows to accommodate this.